frontend_lib/src/component/Timeline/Comment.jsx

```diff
@@ -48,6 +51,31 @@ const Comment = props => {
                 <HTMLContent isTranslated={props.translationState === TRANSLATION_STATE.TRANSLATED}>{props.text}</HTMLContent>
               </div>
             </div>
+
+            <DropdownMenu
+              buttonCustomClass='comment__body__content__actions'
+              buttonIcon='fas fa-ellipsis-v'
+              buttonTooltip={props.t('Actions')}
+            >
+              <button
```

Comment:
I guess you have a reason not to use `IconButton` here? Maybe leave a comment to explain why if it is the case.

Reply:
will fix